src/System.Reflection.Context/dir.props

```diff
@@ -4,6 +4,5 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.3.0</AssemblyVersion>
     <AssemblyKey>ECMA</AssemblyKey>
-    <IsUAP>true</IsUAP>
```

Comment:
does that mean that this package was inbox in uap 10.0.16299?

Reply:
Yes the library was inbox there.

Reply:
Can we somehow overrule the inboxed version which is just a PNSE throwing assembly so that it uses the OOB one instead?